server/protocols/jmap/src/test/java/org/apache/james/jmap/model/OldKeywordTest.java

```diff
@@ -36,6 +36,147 @@ public void shouldRespectBeanContract() {
         EqualsVerifier.forClass(OldKeyword.class).verify();
     }
 
+    @Test
+    public void asKeywordsShouldContainFlaggedWhenIsFlagged() {
+        Optional<OldKeyword> testee = OldKeyword.builder()
+            .isUnread(Optional.empty())
```

Comment:
All the setting at ``Optional.empty()`` are useless

Reply:
True. But that was your Code ;-)

I'll have a fixup on this.

Reply:
Really ?